_posts/2021-08-18-using-http-bridge-as-a-kubernetes-sidecar.md

```diff
@@ -0,0 +1,409 @@
+---
+layout: post
+title:  "Using HTTP Bridge as a Kubernetes sidecar"
+date: 2021-08-18
+author: jakub_scholz
+---
+
+Bridging between Apache Kafka and HTTP protocols is something Kafka users are often asking for.
```

Comment:
is "protocols" (plural) used on purpose referring to both HTTP and HTTPS or just a typo?

Reply:
It is more as in bridging between Apache Kafka (=> Kafka protocol) and HTTP (=> HTTP protocol) ... so two protocols. I did not thought about HTTPS here. Do you think it should be singular here?

Reply:
I guess we can also leave out `protocols` completely and just say _Bridging between Apache Kafka and HTTP is something Kafka users are often asking for._, WDYT?

Reply:
Ok now I got what you meant, but it wasn't clear to me ... I agree on leaving "protocols" out.